_includes/tutorials/rekeying/ksql/code/Makefile

```diff
@@ -11,9 +11,9 @@ tutorial:
 	mkdir $(DEV_OUTPUTS_DIR)
 	mkdir -p $(TEST_OUTPUTS_DIR)
 	harness-runner ../../../../../_data/harnesses/rekeying/ksql.yml $(TEMP_DIR) $(SEQUENCE)
-	bash -c "diff --strip-trailing-cr <(cut -d ',' -f 2- $(STEPS_DIR)/dev/expected-print-input.log|sort) <(cut -d ',' -f 2- $(DEV_OUTPUTS_DIR)/print-input-topic/output-0.log|sort)"
+	bash -c "diff --strip-trailing-cr <(cut -d ',' -f 2-5 $(STEPS_DIR)/dev/expected-print-input.log|sort) <(cut -d ',' -f 2-5 $(DEV_OUTPUTS_DIR)/print-input-topic/output-0.log|sort)"
```

Comment:
This change is to remove the partition of the record.  Since there are no keys inserted, the partition is non-deterministic.